docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -65,7 +65,7 @@ Voluntarily leaving the Steering Committee
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 A Committee member can voluntarily leave the Committee.
-In this case, they notify the other members, create an issue in the `Community Topics <https://github.com/ansible-community/community-topics/issues>`_ repository announcing the resignation, and after that, they are no longer considered Committee members.
+In this case, the member notifies the other members, creates a `Community Topic <https://forum.ansible.com/tags/c/project/7/community-wg>`_ announcing their resignation, and after that, they are no longer considered a Committee member.
```

Comment:
```suggestion
In this case, the member notifies the other members via a PM to the `SteeringCommittee` group in the forum. This change in the steering committee has also to be announced in Bullhorn. If the member voluntarily leaving doesn't want to write this announcement, one of the remaining SC members will write it. After that, they are no longer considered a Committee member.
```

Reply:
LGTM, thanks

Reply:
LGTM too, thanks! how about:
```
In this case, the member notifies the other members via a private message to the `SteeringCommittee` group in the forum. This change in the steering committee has also to be announced in Bullhorn. If the member voluntarily leaving does not want to write this announcement, one of the remaining Committee members will write it. After that, they are no longer considered a Committee member.
```
This contains only formatting things to adhere to the style guide.
@mariolenz if you're OK, could you please edit the suggestion, then i'll "commit" it to the PR

Reply:
@Andersson007 I've edited the suggestion. Is this what you want? Or did I make a C&P error there?

Reply:
@mariolenz yep, committed, thanks